erpnext/templates/pages/help.py

```diff
@@ -5,22 +5,74 @@
 
 def get_context(context):
 	context.no_cache = 1
+	context.bg = 'background-color: #fafbfc; border-radius:0'
+	context.align_greeting = 'start'
+	context.align_search_box = '0'
 	settings = frappe.get_doc("Support Settings", "Support Settings")
 	s = settings
 
+	context.greeting_text = s.greeting_text if s.greeting_text else "We're here to help"
+
+	if s.greeting_text_and_search_bar_alignment == 'Center':
+		context.align_greeting = 'center'
+		context.align_search_box = '25%'
+	if s.greeting_text_and_search_bar_alignment == 'Right':
+		context.align_greeting = 'end'
+		context.align_search_box = '50%'
+	if s.background == 'Color' and s.select_color:
+		context.bg = 'background-color: ' + s.select_color + '; border-radius:0'
+	if s.background == 'Image' and s.add_image:
+		context.bg = 'background-image: url(' + s.add_image + '); background-repeat: no-repeat; border-radius:0'
```

Comment:
This is too hacky, so much configuration is not necessary in the first place. If at all it is required, it should be achieved via CSS classes

Reply:
I wasn't able to figure out a way to use the jinja variable inside the style tag to change the value for a particular class. If you have an idea for it, it would be helpful.